<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>211</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>211</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Calendario">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>139</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>CALENDARIO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actividades">
       <property name="maximumSize">
        <size>
         <width>139</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>ACTIVIDADES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clases">
       <property name="maximumSize">
        <size>
         <width>139</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>CLASES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salir">
       <property name="maximumSize">
        <size>
         <width>139</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>SALIR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>9</y>
      <width>661</width>
      <height>421</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>661</width>
      <height>421</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>391</width>
        <height>291</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>441</width>
        <height>321</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCalendarWidget" name="calendarWidget">
         <property name="maximumSize">
          <size>
           <width>439</width>
           <height>319</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>241</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="lActividades"/>
       </item>
       <item>
        <widget class="QListView" name="lClases"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tClases">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>651</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>114</y>
        <width>451</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QTimeEdit" name="horaInicio"/>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Hora Inicio</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Hora Fin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QTimeEdit" name="horaFin"/>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Tipo de Actividad</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Tarea</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Evaluacion</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Estudio</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Mes</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="dia">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Descripcion</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="mes">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Dia</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="ano">
         <property name="minimum">
          <number>2014</number>
         </property>
         <property name="maximum">
          <number>2020</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>AÃ±o</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="aceptar">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ACEPTAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelar">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>491</width>
        <height>350</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QCheckBox" name="martes">
           <property name="text">
            <string>Martes</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="lunes">
           <property name="text">
            <string>Lunes</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="miercoles">
           <property name="text">
            <string>Miercoles</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QCheckBox" name="jueves">
           <property name="text">
            <string>Jueves</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="viernes">
           <property name="text">
            <string>Viernes</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="sabado">
           <property name="text">
            <string>Sabado</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="nombreProfe"/>
       </item>
       <item row="0" column="1">
        <widget class="QPlainTextEdit" name="nombreClase"/>
       </item>
       <item row="3" column="1">
        <widget class="QTimeEdit" name="inicio">
         <property name="maximumTime">
          <time>
           <hour>19</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="minimumTime">
          <time>
           <hour>7</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>h AP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPlainTextEdit" name="nSalon"/>
       </item>
       <item row="4" column="1">
        <widget class="QTimeEdit" name="fin">
         <property name="maximumTime">
          <time>
           <hour>21</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="minimumTime">
          <time>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>h AP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre Clase</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Hora Inicio</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Hora Fin</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Salon</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nombre Profesor</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Dias</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="cAceptar">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ACEPTAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cCancelar">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CANCELAR</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNueva_Actividad"/>
    <addaction name="actionNueva_Clase"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNueva_Actividad">
   <property name="text">
    <string>Nueva Actividad</string>
   </property>
  </action>
  <action name="actionNueva_Clase">
   <property name="text">
    <string>Nueva Clase</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
